vllm/v1/worker/gpu_worker.py

```diff
@@ -209,6 +229,193 @@ def profile(self, is_start=True):
         else:
             self.profiler.stop()
 
+    def check_health(self) -> None:
+        # worker will always be healthy as long as it's running.
+        return
+
+
+@dataclass
+class WorkerProcHandle:
+    proc: BaseProcess
+    rank: int
+    ready_path: str
+    worker_response_mq: MessageQueue  # The worker process writes to this MQ
+
+
+class WorkerProc:
+    """Wrapper that runs one Worker in a separate process."""
+
+    READY_STR = "READY"
```

Comment:
we should move both classes into `multiproc_executor.py` , or follow the v0 design, to have a `multiproc_worker_utils.py` .

the motivation for `WorkerWrapperBase`, is that we want to do something even before importing the `worker_cls` . if we put both classes here, before `WorkerWrapperBase` is executed, the `worker_cls` is already imported.

Reply:
Moved them. TIL about the import issue